### PR TITLE
Updating Robotics Hub without references to VHACD

### PR DESCRIPTION
**Pull request by vidurvij-Unity**

### CODE REVIEW

File: tutorials/pick_and_place/PickAndPlaceProject/Packages/manifest.json [hunk 1]
```diff
@@ -4,7 +4,7 @@
     "com.unity.ide.rider": "2.0.7",
     "com.unity.ide.visualstudio": "2.0.3",
     "com.unity.ide.vscode": "1.2.2",
-    "com.unity.robotics.urdf-importer": "git+https://github.com/Unity-Technologies/URDF-Importer.git#v0.0.3",
+    "com.unity.robotics.urdf-importer": "git+https://github.com/Unity-Technologies/URDF-Importer.git#v0.0.3-no-vhacd",
```
Comment (hyounesy) [inline tutorials/pick_and_place/PickAndPlaceProject/Packages/manifest.json:7]: should also commit the packages-lock.json.

**Review by hyounesy — CHANGES_REQUESTED** [review]
should also commit the packages-lock.json with the updated "version" and "hash":
```
"version": "git+https://github.com/Unity-Technologies/URDF-Importer.git#v0.0.3-no-vhacd",
...
"hash": "32c1a31bdeebd08db3e6aff511f0f23609c7a82f"
```

**Comment by at669** [comment]
@vidurvij-Unity I'm still running into issues opening the project--might need to clear out [URDF gitmodules](https://github.com/Unity-Technologies/URDF-Importer/blob/vidur/URDF_without_VHACD/.gitmodules) or else it still seems to try to pull the repo

**Comment by peifeng-unity** [comment]
@vidurvij-Unity Also could we update the instruction and let people not use VHACD?
https://github.com/Unity-Technologies/Unity-Robotics-Hub/blob/912e28158b38d3723a69f6df59b9427a9cc9040a/tutorials/pick_and_place/1_urdf.md

**Review by at669 — APPROVED** [review]

**Review by peifeng-unity — APPROVED** [review]

**Review by hyounesy — APPROVED** [review]

**Comment by vidurvij-Unity** [comment]
This PR was meant to be a placeholder for URDF Importer, in case we didn't receive authorization to use VHACD. I think we are are passed that.

**Pull request closed without merging**